spec/index.html

```diff
@@ -6084,6 +6116,89 @@ <h5>sameValue</h5>
             </p>
           </section>
 
+          <section id="func-compareTripleTerm">
+            <h5>compareTripleTerm</h5>
+            <pre class="prototype nohighlight">
+              <span class="return">xsd:integer</span> <span class="operator" style="text-transform: none;">compareTripleTerm</span>(<span class="type"><span class="type RDFterm">Triple term</span></span> <span class="name">term1</span>, <span class="type"><span class="type RDFterm">Triple term</span></span> <span class="name">term2</span>)
+            </pre>
+
+            <p>This function cannot be used directly in expressions. The purpose
+              of this function is to define the semantics of comparison operators,
+              such as "=" and "&lt;",
+              for the case of comparing <a data-cite="RDF12-CONCEPTS#dfn-triple-term">Triple Terms</a>.
+              The function is used in Section <a href="#OperatorMapping" class="sectionRef"></a>.
+            </p>
+
+            <p>This function returns `-1`, `0`, or `1`, or throws an error.
+                Comparison is performed pairwise on the components of the triple terms
+                in the order subject, predicate, object until the result is determined.
+                An error is thrown if comparing two components results in an error.</p>
+
+            <p>Triple term component comparison
+                requires a relative ordering for pairs of IRIs and for pairs of blank nodes,
+                even if no operator extension exists for them
+                in <a href="#operatorExtensibility" class="sectionRef"></a>.
+                The definition of this relative ordering is up to the implementation,
+                as long as it is consistent across query executions.</p>
+
+            <p class="note">For example, the relative ordering of IRIs could behave similarly to
```

Comment:
Sorry, I didn't read it over enough times. This should do it.
```suggestion
            <p class="note">For example, the relative ordering of IRIs here could behave similarly to that seen with
```